### ISSUE TITLE
Need an updated popcorn-complete.js

### ISSUE THREAD

**Issue by steeevio** [mozilla/popcorn-js#467]
The latest version on the website is 1.5.6 but I think the youtube api fixes were put in after that. Is it possible to get version 1.5.11 somewhere? Or how can I go about building my own? Simply including the relevant plugin files and wrappers in my site is not working.

**Comment by jyxia** [comment]
There is Makefile in this repo, so you can `make` it and build you own.

**Comment by NathanLawrence** [comment]
I agree with @jyxia - this should be made as needed to make sure it's not out of sync with the main codebase, not added to the repository.

**Comment by sboy031** [comment]
Why? The project is dead - there should at least be a popcorn-complete.js and popcorn-complete.min.js for all those people who were linking to the now deprecated online versions.
